agent/engine/docker_container_engine.go

```diff
@@ -573,6 +571,12 @@ func (dg *DockerGoClient) ContainerEvents(ctx context.Context) (<-chan DockerCon
 				// These result in us falling through to inspect the container, some
 				// out of caution, some because it's a form of state change
 
+			case "oom":
+				seelog.Infof("process within container %v died due to OOM", event.ID)
```

Comment:
I wish we could have more detail than just `event.ID` here (e.g., the task ARN would be really useful), but I think this is reasonable for now.